concent_api/api-e2e-additional-verification-test.py

```diff
@@ -417,84 +431,21 @@ def test_case_6_test_without_script_src_in(cluster_consts: ProtocolConstants, cl
         price=1000,
     )
 
-    ack_subtask_results_verify = api_request(
-        cluster_url,
-        'send',
-        sci_base.provider_private_key,
-        CONCENT_PUBLIC_KEY,
-        subtask_results_verify,
-        expected_status=200,
-        expected_message_type=message.concents.AckSubtaskResultsVerify,
-        expected_content_type='application/octet-stream',
-    )
-
-    response = upload_file_to_storage_cluster(
-        result_file_content,
-        ack_subtask_results_verify.file_transfer_token.files[0]['path'],  # type: ignore
-        ack_subtask_results_verify.file_transfer_token,  # type: ignore
-        sci_base.provider_private_key,
-        sci_base.provider_public_key,
-        CONCENT_PUBLIC_KEY,
-        STORAGE_CLUSTER_ADDRESS,
-    )
-    assert_condition(response.status_code, 200, 'File has not been stored on cluster')
-    print('\nUploaded file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(
-        subtask_results_verify.task_id,
-        result_file_checksum,
-        result_file_size
-    ))
-
-    response = upload_file_to_storage_cluster(
-        source_file_content,
-        ack_subtask_results_verify.file_transfer_token.files[1]['path'],  # type: ignore
-        ack_subtask_results_verify.file_transfer_token,  # type: ignore
-        sci_base.provider_private_key,
-        sci_base.provider_public_key,
-        CONCENT_PUBLIC_KEY,
-        STORAGE_CLUSTER_ADDRESS,
-    )
-    assert_condition(response.status_code, 200, 'File has not been stored on cluster')
-    print('\nUploaded file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(
-        subtask_results_verify.task_id,
-        source_file_checksum,
-        source_file_size
-    ))
-
-    # Adding calculated number of seconds to time sleep makes us sure that subtask is after deadline.
-    sleep_time = calculate_verification_deadline(
-        subtask_results_verify.subtask_results_rejected.timestamp,
-        cluster_consts.additional_verification_call_time,
-        subtask_results_verify.subtask_results_rejected.report_computed_task.size,
-        cluster_consts.minimum_upload_rate,
-    ) - current_time
-    print(f"Going to sleep for {sleep_time} secs...")
-    time.sleep(
-        sleep_time
-    )
-
-    api_request(
-        cluster_url,
-        'receive',
-        sci_base.requestor_private_key,
-        CONCENT_PUBLIC_KEY,
-        create_client_auth_message(sci_base.requestor_private_key, sci_base.requestor_public_key, CONCENT_PUBLIC_KEY),
-        expected_status=200,
-        expected_message_type=message.concents.SubtaskResultsSettled,
-        expected_content_type='application/octet-stream',
-    )
+    # Setting invalid parameters which are necessary for proper blender work
```

Comment:
Hm... I would rephrase this sentence:
```suggestion
    # Setting parameters which are necessary for proper blender work to invalid values
```

Reply:
Done